03_launch_mgmt_cluster.sh

```diff
@@ -10,6 +10,7 @@ source lib/releases.sh
 # shellcheck disable=SC1091
 source lib/network.sh
 
+export CAPI_CONFIG_FOLDER="${CONFIG_FOLDER}/cluster-api"
```

Comment:
The tests fail to `mkdir -p /cluster-api`  which tells us CONFIG_FOLDER is empty. We don't run `set -eu`, only` set -xe` so it fails to that.